### PR TITLE
Deprecation of ubuntu 16.04 in pipeline 

### PR DESCRIPTION
**Pull request by Aakash-kaushik**
image
The support will go out in September and azure only uses the LTS versions of ubuntu and tags them with `ubuntu-latest` so using this should fix us and wouldn't need any change or a long time but I can also see the benefit of using `ubuntu-20.04` where it doesn't changes and we can have a testing env we know.   

Let me know your opinions.

### CODE REVIEW

File: .ci/ci.yaml [hunk 1]
```diff
@@ -28,7 +28,7 @@ jobs:
 - job: Linux
   timeoutInMinutes: 360
   pool:
-    vmImage: ubuntu-16.04
+    vmImage: ubuntu-latest
```
Comment (zoq) [inline .ci/ci.yaml:31]: Using `ubuntu-latest` sounds reasonable to me.

Comment (Aakash-kaushik) [reply .ci/ci.yaml:31]: Nice, also this is the same for mlpack too, so should i open a PR there too ?

Comment (zoq) [reply .ci/ci.yaml:31]: For mlpack we wanted to make sure we run tests against the oldest version, so we will wait until September 20.

Comment (Aakash-kaushik) [reply .ci/ci.yaml:31]: Sure, maybe we can leave it that way.

**Comment by Aakash-kaushik** [comment]
All the checks do pass so maybe we can merge this ?

**Review by zoq — APPROVED** [review]
Looks great to me, thanks for the update.

**Comment by zoq** [comment]
Thanks again for the patch!

**Pull request merged**